web-console/future-openshift/dashboards/dashboards.md

```diff
@@ -0,0 +1,34 @@
+# Dashboards
+
+## Dashboard Organization
+
+### Dashboard-related navigation changes
+![Dashboard navigation](img/dashboard-navigation.png)
+- The `Status` navigation item will be removed and replaced with `Dashboards`.
+- If technically feasible, `Dashboards` should be the landing page of OpenShift and should be the first navigation item, followed by `Projects`. If not, the order should remain unchanged.
+- The status page for "All Projects" will be removed and replaced with the new cluster scoped overview dashboard
+- Project specific status pages will become tabs of a detail view for each project. See "Object scoped dashboards" below.
+
+### Cluster scoped dashboards
+![Dashboard navigation](img/cluster-scoped.png)
+
+#### Overview dashboard
+- The `Overview` dashboard is the primary tab on the dashboards page and should provide high-level information about the cluster and its components. Specific notes on the content of this dashboard are found elsewhere.
+
+#### Specialized dashboards
+- Additional dashboard tabs may be added for specific parts of OpenShift or potentially by Operators. These should be placed to the right of the `Overview` dashboard tab and can be organized as necessary for the relevant information
+
+### Object scoped dashboards
+- The `Overview` tab on object details pages should primarily be used to illustrate the YAML of the object in a visual way. If applicable, a few select charts with important metrics may be included on the page.
+- If the metrics charts would take up more than a full row on a desktop-sized screen, a separate `Dashboard` tab should be added and any charts should be placed there instead of on the `Overview` tab.
```

Comment:
There's probably more nuance involved in deciding whether or not an object should have a `Dashboard` tab than its desktop layout or its usage of "metrics charts" specifically. 7 utilization metrics, for example, could all be combined into a single 2-column card and not match this rule. A Health card also wouldn't meet this rule, but I'd generally expect that card to only appear within a `Dashboard`.

I'm fine with this as a general guideline for now, but as we dig deeper into how these cards could be used elsewhere (even within `Overview`) this decision may become more of an art than a quantifiable science. We'll see though!